.github/workflows/nia-e2e.yaml

```diff
@@ -0,0 +1,49 @@
+name: Now in Android E2E
+
+on:
+  workflow_dispatch:
+  pull_request:
+    types: [opened, reopened, synchronize, labeled]
+    branches:
+      - main
+    paths:
+      - '.github/workflows/nia-*'
+      - 'nowinandroid/**'
+
+permissions:
+  contents: read
+  
+jobs:
+  nia-e2e:
+    name: Setup Anbox and Run E2E Tests
+    runs-on: ubuntu-latest
+    if: contains(github.event.pull_request.labels.*.name, 'run-nia-e2e-tests')
+    timeout-minutes: 60
+    defaults:
+      run:
+        working-directory: ./nowinandroid
```

Comment:
If you set a default working-directory I don't think the `uses: ./.github/...` below will work.